docs/merkletree/treetex/main.go

```diff
@@ -318,6 +326,32 @@ func main() {
 	flag.Parse()
 	height := uint(bits.Len64(*treeSize-1)) + 1
 
+	var innerNodeText = func(id compact.NodeID) string {
+		if id.Level == 0 {
+			panic("attempting to use inner node text func for leaf text")
+		}
```

Comment:
I don't think these 3 lines are necessary.

Reply:
They got dinged already (although it looks like I got distracted and didn't push the updated branch :/)